document_page/models/document_page.py

```diff
@@ -144,51 +151,9 @@ def write(self, vals):
     @api.model
     @api.returns('self', lambda value: value.id)
     def create(self, vals):
-        page_id = super(document_page, self).create(vals)
+        """Create the first history of a document."""
```

Comment:
Hi @dreispt. 
Please i don't understand this remark. There is no result variable in this method. Please explain me what you talking about here